source/vir/src/sst_to_air.rs

```diff
@@ -868,24 +884,8 @@ pub(crate) fn exp_to_expr(ctx: &Ctx, exp: &Exp, expr_ctxt: &ExprCtxt) -> Result<
                 BinaryOp::Bitwise(bo) => {
                     let box_lh = try_box(ctx, lh, &lhs.typ).expect("Box");
                     let box_rh = try_box(ctx, rh, &rhs.typ).expect("Box");
-                    let width_left = match bitwidth_from_type(&lhs.typ) {
-                        Some(w) => w,
-                        None => {
-                            return err_string(
-                                &lhs.span,
-                                format!("error: unable to get bit-width from type {:?}", lhs.typ),
-                            );
-                        }
-                    };
-                    let width_right = match bitwidth_from_type(&rhs.typ) {
-                        Some(w) => w,
-                        None => {
-                            return err_string(
-                                &rhs.span,
-                                format!("error: unable to get bit-width from type {:?}", rhs.typ),
-                            );
-                        }
-                    };
+                    let width_left = bitwidth_from_type(&lhs.typ).expect("bounded integer type");
+                    let width_right = bitwidth_from_type(&rhs.typ).expect("bounded integer type");
```

Comment:
I guess this will not actually panic since we have the check at beginning of `ExpX::Binary` case (line 768). Am I right? 
```
line 768: if !allowed_bitvector_type(&exp.typ) {
```

Reply:
no, these 2 lines aren't part of that case. However these should pass because we don't define bitwise ops for ints or nats in the first place.